components/automate-ui/src/app/modules/user/user-management/user-management.component.ts

```diff
@@ -119,12 +146,12 @@ export class UserManagementComponent implements OnInit, OnDestroy {
     this.store.dispatch(new DeleteUser(this.userToDelete));
   }
 
-  public handleCreateUser(): void {
+  public createUser(): void {
     this.creatingUser = true;
     const formValues = this.createUserForm.value;
 
     const userCreateReq = <CreateUserPayload>{
```

Comment:
Since you're here, a bit of cleanup: much better to declare types rather than cast (casting in TypeScript can often be misleading).
```suggestion
    const userCreateReq: CreateUserPayload = {
```